mage_integrations/mage_integrations/sources/salesforce/client/tap_salesforce/sync.py

```diff
@@ -53,14 +44,9 @@ def get_stream_version(catalog_entry, state):
         return stream_version
     return int(time.time() * 1000)
 
-
 def resume_syncing_bulk_query(sf, catalog_entry, job_id, state, counter):
     bulk = Bulk(sf)
-    current_bookmark = singer.get_bookmark(state, catalog_entry['tap_stream_id'],
-                                           'JobHighestBookmarkSeen') or sf.get_start_date(
-                                                                                    state,
-                                                                                    catalog_entry
-                                                                                         )
+    current_bookmark = singer.get_bookmark(state, catalog_entry['tap_stream_id'], 'JobHighestBookmarkSeen') or sf.get_start_date(state, catalog_entry)
```

Comment:
this line is too long. can you lint the files with flake8?

Reply:
Linted in 8cc2a0e2931ffc6e60ebd0611b0fa1521525ccda